lib/theme_template/example/index.html

```diff
@@ -5,7 +5,7 @@
 Lorem ipsum dolor sit amet, quo id prima corrumpit pertinacia, id ius dolor dolores, an veri pertinax explicari mea. Agam solum et qui, his id ludus graeco adipiscing. Duis theophrastus nam in, at his vidisse atomorum. Tantas gloriatur scripserit ne eos. Est wisi tempor habemus at, ei graeco dissentiet eos. Ne usu aliquip sanctus conceptam, te vis ignota animal, modus latine contentiones ius te.
```

Comment:
I have a comment above that says `# Specify a layout from your theme!` so users don't feel confined to `default` or `page` or whatever. Should we just choose one anyway?